charts/k8ssandra/templates/cassandra/cassdc.yaml

```diff
@@ -55,19 +55,19 @@ spec:
 {{- if .Values.cassandra.auth.superuser.secret }}
   superuserSecretName: {{ .Values.cassandra.auth.superuser.secret }}
 {{- else if .Values.cassandra.auth.superuser.username }}
-  superuserSecretName: {{ include "k8ssandra.clusterName" . }}-superuser
+  superuserSecretName: {{ .Release.Name }}-{{ include "k8ssandra.clusterName" . }}-superuser
```

Comment:
@burmanm 
As reference, we are now putting `Release.Name` as the cluster name when not specified per [this release 1.0 change](https://github.com/k8ssandra/k8ssandra/blob/5041f1419f871cbc0506a15bf45b9a4b28771c16/charts/k8ssandra/templates/_helpers.tpl#L10).

So, in this case, we are looking at `myreleaseX-myreleaseX-superuser`.

Is this the intent here for this change?

Reply:
Wanting to make sure you know that the release name is now the clusterName.

Reply:
>Is this the intent here for this change?

@jeffbanks That is not the intent of the change. When I created #335. I was of the mindset that I should be able to install the k8ssandra chart multiple times in different namespaces using the same release name. It doesn't work though. This PR was a short spike to see what changes are necessary. Unfortunately, it becomes non-trivial and I think will be easier to address with an operator-based implementation.

Reply:
This PR only showed which parts of the stack do not correctly function with more than one release in a single namespace. And although one can now install it - it will blow up your cluster with high load so in reality it can't be done.

Reply:
But we could still add those missing releaeNames to the grafana parts and secrets, even if we don't recommend killing the cluster.